### DISCUSSION TITLE
Vote for Fedomn as RisingLight member

### DISCUSSION THREAD

**Discussion by skyzh** [risinglightdb/risinglight#484] [Member]
@Fedomn has made great contributions to the RisingLight project in the past month. Here's a list of their pull requests: https://github.com/risinglightdb/risinglight/pulls?q=author%3AFedomn

@Fedomn has submitted various fixes and improvements in RisingLight, where the most significant one is https://github.com/risinglightdb/risinglight/pull/447. In this PR, they modified the RowSetIterator to scan RowHandler only, which significantly reduced I/O required for queries like `select count(*) from table`.

@Fedomn demonstrated a good understanding of the storage part of RisingLight project. Therefore, I'd like to promote them as RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. According to [GOVERNANCE](https://github.com/risinglightdb/risinglight/blob/main/GOVERNANCE.md), a minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by skyzh** [comment]
With 3 votes from @skyzh @zzl200012 @likg227, we are accepting this vote. Welcome to the RisingLight community! The invitation email should have been sent. And @Fedomn will be granted write access to all our repos.

**Comment by Fedomn** [comment]
Thanks everyone
